tests/test_highlevel_extracttext.py

```diff
@@ -60,6 +69,26 @@ def test_simple4_with_string(self):
         s = run_with_string(test_file)
         self.assertEqual(s, test_strings[test_file])
 
+    def test_simple1_with_pathlib(self):
+        test_file = "simple1.pdf"
+        s = run_with_pathlib(test_file)
+        self.assertEqual(s, test_strings[test_file])
+
+    def test_simple2_with_pathlib(self):
+        test_file = "simple2.pdf"
+        s = run_with_pathlib(test_file)
+        self.assertEqual(s, test_strings[test_file])
+
+    def test_simple3_with_pathlib(self):
+        test_file = "simple3.pdf"
+        s = run_with_pathlib(test_file)
+        self.assertEqual(s, test_strings[test_file])
+
+    def test_simple4_with_pathlib(self):
+        test_file = "simple4.pdf"
+        s = run_with_pathlib(test_file)
+        self.assertEqual(s, test_strings[test_file])
+
```

Comment:
No need to add these end-to-end tests for this feature. It's already properly tested by the similiar tests. Could you add tests specifically for `absolute_sample_path` instead?

Reply:
Tests for `absolute_sample_path`? Would you give a code example? I'm a little bit confused.

Reply:
Do you mean tests for `open_filename` like 4ece1faa557838786c0e6f54a75199bfaa9c5336? (not pushed to develop yet)

Reply:
Yes. Test the happy flow.

Reply:
Fixed in 575078b.